modules/storages/app/components/storages/admin/forms/automatically_managed_project_folders_form_component.rb

```diff
@@ -94,8 +94,8 @@ def storage_provider_credentials_copy_instructions
     def provider_credentials_instructions_link
       render(
         Primer::Beta::Link.new(
-          href: Storages::Peripherals::StorageInteraction::Nextcloud::Util.join_uri_path(storage.host,
-                                                                                         "settings/admin/openproject"),
+          href: Storages::Peripherals::StorageInteraction::RequestUrlBuilder.build(storage,
```

Comment:
I am not suggesting to do it here.
In my opinion the following namespace is absolutely sufficient from code user perspective. Therefore let's remove this two levels of nesting `::Peripherals::StorageInteraction`. Wdyt?
```suggestion
          href: Storages::RequestUrlBuilder.build(storage,
```

Reply:
Hmm, I don't understand. Removing those namespace identifiers will lead to the `RequestUrlBuild` not being found?

Reply:
Sorry for not being clear enough. I meant removing this 2 levels entirely throughout our application, because they do not much meaning, but using them makes names enormous. And wanted to know your opinion about that. Do they bring value to you?

Reply:
The namespace means value for me due to its context.

e.g. a `Utils` class within `StorageInteraction::Nextcloud` means it is meant to be used there. Hence, when long namespace definitions like here are seen, then this is a good indicator, that the context is wrong. Which is, what you raised here.

So, When I was planning this url build, I opted for the commands/queries. When I replaced the previous utils method, I noticed, that we need this functionality outside, too. (here in the form components). With this information I agree with you, we should move it up.